src/SFA.DAS.RoatpOversight.Web/Startup.cs

```diff
@@ -180,6 +180,8 @@ private void ConfigureDependencyInjection(IServiceCollection services)
             services.AddTransient<IApplicationOutcomeOrchestrator, ApplicationOutcomeOrchestrator>();
             services.AddTransient<IRoatpRegisterTokenService, RoatpRegisterTokenService>();
             services.AddTransient<IOversightOrchestrator, OversightOrchestrator>();
+            services.AddTransient<IAppealOrchestrator, AppealOrchestrator>();
+            services.AddTransient<IPdfValidatorService, PdfValidatorService>();
```

Comment:
Am I correct in thinking this could be a Singleton instead of Transient?

Reply:
I believe you may be right, will change